doc/source/whatsnew/v0.25.0.rst

```diff
@@ -275,7 +275,7 @@ Numeric
 Conversion
 ^^^^^^^^^^
 
--
+- Bug in :func:`DataFrame.astype()` when passing a dict of columns and types the `errors` parameter was discarded by mistake. (:issue:`25905`)
```

Comment:
'was discarded by mistake' -> 'was ignored.'

Reply:
Changed to this in 1c35cc70108a0407d964c26b5cfc07d654dd4ebc